Update fungible token to 0.2.0 (#80)
nearprotocol/NEPs: specs/Standards/Tokens/FungibleToken.md

Note, that the precision is not part of the default standard, since it's not required to perform actions. The minimum
value is always 1 token.

The standard acknowledges NEAR storage staking model and accounts for the difference in storage that can be introduced
by actions on this contract. Since multiple users use the contract, the contract has to account for potential

> storage increase. That's why every change method of the contract that can change the amount of storage has to be payable.

Reviewer: ```suggestion
storage increase. Thus every change method of the contract that can change the amount of storage must be payable.
```

Just curious, why can't this cost be covered by gas?  Since it's variable?

Reviewer: Also if a method only decreases state why would an attached deposit be needed?

Author: It's required to be part of the account balance. Since the gas can only be used to write into the storage, but not to permanently store it.

## Reference-level explanation
[reference-level-explanation]: #reference-level-explanation

> The full implementation in Rust can be found there: [fungible-token](https://github.com/nearprotocol/near-sdk-rs/blob/master/examples/fungible-token/src/lib.rs)

Reviewer: ```suggestion
The full implementation in Rust can be found here: [fungible-token](https://github.com/nearprotocol/near-sdk-rs/blob/master/examples/fungible-token/src/lib.rs)
```

**NOTES:**
- All amounts, balances and allowance are limited by `U128` (max value `2**128 - 1`).
- Token standard uses JSON for serialization of arguments and results.
- Amounts in arguments and results have are serialized as Base-10 strings, e.g. `"100"`. This is done to avoid
JSON limitation of max integer value of `2**53`.
- The contract tracks the change in storage before and after the call. If the storage increases,
the contract requires the caller of the contract to attach enough deposit to the function call
to cover the storage cost.

> This is done to prevent a denial of service attack on the contract by taking all available storage.

Reviewer: This seems a little vague.  Perhaps mention that since the cost of adding new escrow account is cheap, many accounts can be created until the contract runs out of stroage.

/// * If this function is called by an escrow account (`owner_id != predecessor_account_id`),
///   then the allowance of the caller of the function (`predecessor_account_id`) on
///   the account of `owner_id` should be greater or equal than the transfer `amount`.
/// * Caller of the method has to attach deposit enough to cover storage difference at the
///   fixed storage price defined in the contract.
#[payable]

> pub fn transfer_from(&mut self, owner_id: AccountId, new_owner_id: AccountId, amount: U128);

Reviewer: How does this function lead to an increase in state?

Author: If the account for the new owner is created
